c_api/IndexIVFFlat_c.cpp

```diff
@@ -25,6 +25,8 @@ DEFINE_INDEX_DOWNCAST(IndexIVFFlat)
 DEFINE_GETTER(IndexIVFFlat, size_t, nlist)
 /// number of probes at query time
 DEFINE_GETTER(IndexIVFFlat, size_t, nprobe)
+DEFINE_SETTER(IndexIVFFlat, size_t, nprobe)
```

Comment:
Not sure this setter is very safe to use, but for consistency with C++ it can be added.

Reply:
I can change this setter